packages/logs/src/domain/logsCollection/runtimeError/runtimeErrorCollection.ts

```diff
@@ -13,15 +13,15 @@ export interface ProvidedError {
   handlingStack: string
 }
 
-export function startRuntimeErrorCollection(
-  configuration: LogsConfiguration,
-  lifeCycle: LifeCycle,
-  rawErrorObservable = new Observable<RawError>()
-) {
-  if (configuration.forwardErrorsToLogs) {
-    trackRuntimeError(rawErrorObservable)
+export function startRuntimeErrorCollection(configuration: LogsConfiguration, lifeCycle: LifeCycle) {
+  if (!configuration.forwardErrorsToLogs) {
+    return { stop: noop }
   }
 
+  const rawErrorObservable = new Observable<RawError>()
+
+  trackRuntimeError(rawErrorObservable)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** Not related to your change, but while we are at it, ‏we might want to stop tracking runtime error in the returned `stop()` function